features/guidance/collapse.feature

```diff
@@ -353,13 +353,13 @@ Feature: Collapse
             | ge    | primary | second | no     |
 
         When I route I should get
-            | waypoints | route               | turns                        |
-            | d,c       | first,first,first   | depart,continue uturn,arrive |
-            | a,f       | first,first,first   | depart,continue uturn,arrive |
-            | a,g       | first,second,second | depart,turn left,arrive      |
-            | d,g       | first,second,second | depart,turn right,arrive     |
-            | g,f       | second,first,first  | depart,turn right,arrive     |
-            | g,c       | second,first,first  | depart,turn left,arrive      |
+            | waypoints | route               | turns                          |
+            | d,c       | first,first,first   | depart,continue uturn,arrive   |
+            | a,f       | first,first,first   | depart,continue uturn,arrive   |
+            | a,g       | first,second,second | depart,turn left,arrive        |
+            | d,g       | first,second,second | depart,turn right,arrive       |
+            | g,f       | second,first,first  | depart,turn right,arrive       |
+            | g,c       | second,first,first  | depart,end of road left,arrive |
```

Comment:
the turn is at `b`. This currently oscillates quite heavily due to https://github.com/Project-OSRM/osrm-backend/issues/3365. It all depends on whether the turn at the end is recognised as obvious/not. Since I have fixed some minor errors in angle detection, this now again is recognised as a turn at the end of the road (at `b`)